api/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id "com.diffplug.spotless" version "6.9.1"
+}
+
+dependencies {
+    implementation 'org.eclipse.microprofile.openapi:microprofile-openapi-api:3.0'
+    testImplementation 'org.slf4j:slf4j-jdk14:2.0.0'
+    testImplementation 'org.openapitools.empoa:empoa-simple-models-impl:2.0.0'
+    testImplementation 'nl.jqno.equalsverifier:equalsverifier:3.10.1'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
+    implementation group: 'org.reflections', name: 'reflections', version: '0.10.2'
+}
+
+java {
+    withJavadocJar()
+    withSourcesJar()
+}
+
+spotless {
+    java {
+        removeUnusedImports()
+        trimTrailingWhitespace()
+        palantirJavaFormat()
+    }
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            artifactId = 'openapi-style-validator-api'
+            pom {
+                name = 'OpenAPI Style Validator - api'
+                description = 'module to expose API for writing custom rules to validate the style and standards of an OpenAPI specification'
+                packaging = 'jar'
+                url = 'https://openapitools.github.io/openapi-style-validator/'
+                inceptionYear = "2019"
+                licenses {
+                    license {
+                        name = 'Apache 2.0 License'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0'
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'JFCote'
+                        name = 'Jean-Francois Cote'
+                        email = 'jcote@stingray.com'
```

Comment:
This email is not good anymore. Please remove the line entirely because if possible

Reply:
we can'r remove this as this it required by `publishing` action. I changed it to my details. Let me know if that's ok.